applications/nrf_desktop/src/modules/Kconfig.dfu_mcumgr

```diff
@@ -9,20 +9,16 @@ menuconfig DESKTOP_DFU_MCUMGR_ENABLE
 	select EXPERIMENTAL
 	select CAF_BLE_SMP_TRANSFER_EVENTS if MCUMGR_TRANSPORT_BT
 	select MCUMGR
-	select MCUMGR_GRP_IMG
 	select MCUMGR_GRP_OS
 	select MCUMGR_MGMT_NOTIFICATION_HOOKS
 	select MCUMGR_GRP_IMG_MUTEX if DESKTOP_DFU_LOCK
 	select MCUMGR_SMP_COMMAND_STATUS_HOOKS
 	select NET_BUF
 	select ZCBOR
 	select CRC
-	select IMG_MANAGER
-	select MCUBOOT_BOOTUTIL_LIB
 	select STREAM_FLASH
```

Comment:
Is stream flash used by SUIT?  (please also double check the remaining Kconfig options selected as common)